packages/react-native/React/Views/RCTCursor.m

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// [macOS]
```

Comment:
Not important, just curious what this comment does... Since we have `#if TARGET_OS_OSX` later in the file...

Reply:
Convention that the entire file is owned by us. See https://github.com/microsoft/react-native-macos/blob/main/docs/DiffsWithUpstream.md